src/AzureAuth/Commands/CommandAad.cs

```diff
@@ -21,6 +21,8 @@ namespace Microsoft.Authentication.AzureAuth.Commands
     using Microsoft.Office.Lasso.Interfaces;
     using Microsoft.Office.Lasso.Telemetry;
 
+    using ModeExtensions = Microsoft.Authentication.AzureAuth.AuthModeExtensions;

```

Comment:
Is this still needed?

Reply:
Yes because we have a naming conflict between us and MSALWrapper.